mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -283,6 +283,39 @@
     <!-- Text for the button to not request notification permission on the device and dismiss the dialog -->
     <string name="onboarding_home_enable_notifications_negative_button">Not now</string>
 
+    <!-- Juno first user onboarding flow experiment -->
+    <!-- Title for set firefox as default browser screen.
+        The first parameter is the name of the app defined in app_name (for example: Fenix) -->
+    <string name="juno_onboarding_default_browser_title" tools:ignore="UnusedResources">Make %s your go-to browser</string>
+    <!-- Description for set firefox as default browser screen.
+        The first parameter is the string with key "juno_onboarding_default_browser_description_link_text", that says "privacy notice" in en-* -->
```

Comment:
Since "privacy notice" will appear as translated, should the comment *not* mention "en-*" at the end? And rather something like "(...) that says "privacy notice", and that is possible to translate in another string"

Reply:
sounds good! 👍